pkg/instance/handler.go

```diff
@@ -192,29 +192,31 @@ func (h Handler) Deploy(c *gin.Context) {
 	c.JSON(http.StatusCreated, instance)
 }
 
-func convertRequiredParameters(instanceID uint, requestParameters []ParameterRequest) []model.InstanceRequiredParameter {
+func convertRequiredParameters(instance *model.Instance, requestParameters []ParameterRequest) []model.InstanceRequiredParameter {
 	if len(requestParameters) > 0 {
 		var parameters = make([]model.InstanceRequiredParameter, len(requestParameters))
 		for i, parameter := range requestParameters {
+			stackRequiredParameter := model.StackRequiredParameter{ID: parameter.StackParameterID}
```

Comment:
I would inline this (and in function below) into the struct literal if not used anywhere else. saves us naming/reading/maintaining a var in our head 😋

Reply:
Done!